src/Carbon/CarbonPeriod.php

```diff
@@ -420,6 +420,9 @@ protected static function makeCarbonInterval($var)
      */
     protected static function intervalHasTime(DateInterval $interval)
     {
+        // array_key_exists + get_object_vars check actual microsecond support as public available property
+        // isset or property_exists will both fail on PHP 7.0.14 - 7.0.21 due to the following bug:
+        // https://bugs.php.net/bug.php?id=74852
```

Comment:
Same bug seems to happen as well on 7.1.0 - 7.1.7 but it is not apparent because `f` property exists - https://3v4l.org/tqJaM#v710.

Reply:
Yep it's an other one: https://bugs.php.net/bug.php?id=74852
But the array_key_exists will not be impacted by this one.